hack/prepare-operatorhub-pr.sh

```diff
@@ -1,19 +1,19 @@
-#!/usr/bin/env -S bash -e
+#!/bin/bash -e
 
 # Required CLIs
 # - https://github.com/opencontainers/umoci
 # - https://github.com/containers/skopeo
 
 # Usage:
-# prepare-operatorhu-pr.sh <version> <bundle-image-ref>
+# prepare-operatorhu-pr.sh <version> <bundle-image-ref> <tls-verification-requirement>
 
 TMP_OCI_PATH=$(mktemp -d out/sbo-bundle-oci.XXX)
 
-skopeo copy docker://$2 oci:$TMP_OCI_PATH:bundle
+skopeo copy --src-tls-verify=$3 --dest-tls-verify=$3 docker://$2 oci:$TMP_OCI_PATH:bundle
```

Comment:
`--dest-tls-verify` option is not needed, because the destination is the local folder.

Did you try to turn off tls verification for all cases? That should work and then we do not need to pass extra argument to the script, nor declare the var in Makefile.